1/12/2017 10:17:17 PM   The given key was not present in the dictionary.  mscorlib     at System.Collections.Concurrent.ConcurrentDictionary`2.get_Item(TKey key)
   at NusysServer.CreateNewLibraryElementRequestHandler.HandleRequest(Request request, NuWebSocketHandler senderHandler) in C:\Users\Trent\Documents\nusys_repo\main\nusys\NusysServer\RequestHandlers\CreateNewLibraryElementRequestHandler.cs:line 41
   at NusysServer.CreateNewContentRequestHandler.HandleRequest(Request request, NuWebSocketHandler senderHandler) in C:\Users\Trent\Documents\nusys_repo\main\nusys\NusysServer\RequestHandlers\CreateNewContentRequestHandler.cs:line 48
   at NusysServer.RequestRouter.<HandleRequest>d__0.MoveNext() in C:\Users\Trent\Documents\nusys_repo\main\nusys\NusysServer\Static_Classes\RequestRouter.cs:line 134
1/12/2017 10:17:17 PM   The given key was not present in the dictionary.  mscorlib     at System.Collections.Concurrent.ConcurrentDictionary`2.get_Item(TKey key)
   at NusysServer.CreateNewLibraryElementRequestHandler.HandleRequest(Request request, NuWebSocketHandler senderHandler) in C:\Users\Trent\Documents\nusys_repo\main\nusys\NusysServer\RequestHandlers\CreateNewLibraryElementRequestHandler.cs:line 41
   at NusysServer.CreateNewContentRequestHandler.HandleRequest(Request request, NuWebSocketHandler senderHandler) in C:\Users\Trent\Documents\nusys_repo\main\nusys\NusysServer\RequestHandlers\CreateNewContentRequestHandler.cs:line 48
   at NusysServer.RequestRouter.<HandleRequest>d__0.MoveNext() in C:\Users\Trent\Documents\nusys_repo\main\nusys\NusysServer\Static_Classes\RequestRouter.cs:line 134
1/12/2017 10:18:49 PM   test!    
1/12/2017 10:43:19 PM   test!    
1/12/2017 10:45:19 PM   The remote server returned an error: (404) Not Found.  System     at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at NusysServer.WebSearchRequestHandler.<MakeContentMessage>d__2.MoveNext() in C:\Users\Trent\Documents\nusys_repo\main\nusys\NusysServer\RequestHandlers\WebSearchRequestHandler.cs:line 90
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at NusysServer.WebSearchRequestHandler.<RunParser>d__1.MoveNext() in C:\Users\Trent\Documents\nusys_repo\main\nusys\NusysServer\RequestHandlers\WebSearchRequestHandler.cs:line 57
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at NusysServer.WebSearchRequestHandler.<>c__DisplayClass0_0.<<HandleArgsRequest>b__0>d.MoveNext() in C:\Users\Trent\Documents\nusys_repo\main\nusys\NusysServer\RequestHandlers\WebSearchRequestHandler.cs:line 32
1/12/2017 10:46:59 PM   test!    
1/12/2017 10:54:53 PM   The remote name could not be resolved: 'stateparks'  System     at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at NusysServer.WebSearchRequestHandler.<MakeContentMessage>d__2.MoveNext() in C:\Users\Trent\Documents\nusys_repo\main\nusys\NusysServer\RequestHandlers\WebSearchRequestHandler.cs:line 90
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at NusysServer.WebSearchRequestHandler.<RunParser>d__1.MoveNext() in C:\Users\Trent\Documents\nusys_repo\main\nusys\NusysServer\RequestHandlers\WebSearchRequestHandler.cs:line 57
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at NusysServer.WebSearchRequestHandler.<>c__DisplayClass0_0.<<HandleArgsRequest>b__0>d.MoveNext() in C:\Users\Trent\Documents\nusys_repo\main\nusys\NusysServer\RequestHandlers\WebSearchRequestHandler.cs:line 32
1/12/2017 10:59:02 PM   The underlying connection was closed: An unexpected error occurred on a send.  System     at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at NusysServer.WebSearchRequestHandler.<MakeContentMessage>d__2.MoveNext() in C:\Users\Trent\Documents\nusys_repo\main\nusys\NusysServer\RequestHandlers\WebSearchRequestHandler.cs:line 90
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at NusysServer.WebSearchRequestHandler.<RunParser>d__1.MoveNext() in C:\Users\Trent\Documents\nusys_repo\main\nusys\NusysServer\RequestHandlers\WebSearchRequestHandler.cs:line 57
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at NusysServer.WebSearchRequestHandler.<>c__DisplayClass0_0.<<HandleArgsRequest>b__0>d.MoveNext() in C:\Users\Trent\Documents\nusys_repo\main\nusys\NusysServer\RequestHandlers\WebSearchRequestHandler.cs:line 32
1/12/2017 11:00:21 PM   test!    
1/12/2017 11:09:22 PM   test!    
1/12/2017 11:11:32 PM   The remote server returned an error: (400) Bad Request.  System     at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at NusysServer.WebSearchRequestHandler.<MakeContentMessage>d__2.MoveNext() in C:\Users\Trent\Documents\nusys_repo\main\nusys\NusysServer\RequestHandlers\WebSearchRequestHandler.cs:line 91
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at NusysServer.WebSearchRequestHandler.<RunParser>d__1.MoveNext() in C:\Users\Trent\Documents\nusys_repo\main\nusys\NusysServer\RequestHandlers\WebSearchRequestHandler.cs:line 57
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at NusysServer.WebSearchRequestHandler.<>c__DisplayClass0_0.<<HandleArgsRequest>b__0>d.MoveNext() in C:\Users\Trent\Documents\nusys_repo\main\nusys\NusysServer\RequestHandlers\WebSearchRequestHandler.cs:line 32
1/12/2017 11:14:40 PM   test!    
1/12/2017 11:17:32 PM   The remote server returned an error: (404) Not Found.  System     at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at NusysServer.WebSearchRequestHandler.<MakeContentMessage>d__2.MoveNext() in C:\Users\Trent\Documents\nusys_repo\main\nusys\NusysServer\RequestHandlers\WebSearchRequestHandler.cs:line 93
1/12/2017 11:17:32 PM   The remote server returned an error: (404) Not Found.  System     at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at NusysServer.WebSearchRequestHandler.<MakeContentMessage>d__2.MoveNext() in C:\Users\Trent\Documents\nusys_repo\main\nusys\NusysServer\RequestHandlers\WebSearchRequestHandler.cs:line 93
1/12/2017 11:19:17 PM   test!    
1/12/2017 11:20:33 PM   The remote server returned an error: (404) Not Found.  System     at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at NusysServer.WebSearchRequestHandler.<MakeContentMessage>d__2.MoveNext() in C:\Users\Trent\Documents\nusys_repo\main\nusys\NusysServer\RequestHandlers\WebSearchRequestHandler.cs:line 94
1/12/2017 11:20:49 PM   The remote server returned an error: (404) Not Found.  System     at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at NusysServer.WebSearchRequestHandler.<MakeContentMessage>d__2.MoveNext() in C:\Users\Trent\Documents\nusys_repo\main\nusys\NusysServer\RequestHandlers\WebSearchRequestHandler.cs:line 94
1/12/2017 11:20:52 PM   The remote server returned an error: (404) Not Found.  System     at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at NusysServer.WebSearchRequestHandler.<MakeContentMessage>d__2.MoveNext() in C:\Users\Trent\Documents\nusys_repo\main\nusys\NusysServer\RequestHandlers\WebSearchRequestHandler.cs:line 94
1/12/2017 11:26:05 PM   test!    
